knowledge-base/grid-customize-hierarchy-expand-column-blazor-grid.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Customize Hierarchy Expand Column in Blazor Grid
```

Comment:
I would use 1-2 more specific and relevant words in the titles instead of customize. For example,
```suggestion
title: Reorder, Lock or Resize the Hierarchy Expand Column in a Blazor Grid
```